ticketutil/bugzilla.py

```diff
@@ -1,11 +1,24 @@
 import logging
+import os
 
 import requests
 
+import base64
```

Comment:
This isn't used anywhere. You can remove for this PR (I think you might use this in a future PR for add_attachment()?)